app/views/macro/error-summary.njk

```diff
@@ -0,0 +1,18 @@
+{% from "govuk/components/error-summary/macro.njk" import govukErrorSummary %}
+
+{% macro errorSummary(params) %}
```

Comment:
Really neat util, this kind of pattern could really clean up the many different ways we have of doing this